orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -101,17 +107,18 @@ class ORTTrainer(object):
                                                              { 'params' : ['model_param1' , 'model_param_2'], 'alpha' : 0.0}
                                                            ],
                                             alpha=0.9, beta=0.999)
-            ort_trainer = ORTTrainer(model, model_desc, optim_config)
+            ort_trainer = ORTTrainer(model, model_desc, optim_config, loss_fn)
     """
+    DEFAULT_OPSET_VERSION = 12
```

Comment:
we are not using this global anymore. you can find opset on `ORTTrainer.options._internal_use.opset_version`